treeherder/model/migrations/0009_remove_matcher_table.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-06 09:25
```

Comment:
This migration will need renaming and the `dependencies` property updating, since other migrations have landed since

Reply:
Shoot, my bad.  I'm out of practice on the django side...  :(